src/solc_0.7/common/BaseWithStorage/WithMinter.sol

```diff
@@ -14,13 +14,13 @@ contract WithMinter {
     /// @dev change the minter to be `newMinter`.
     /// @param newMinter address of the new minter.
     function changeMinter(address newMinter) external {
-        require(msg.sender == _minter, "only admin can change admin");
+        require(msg.sender == _minter, "MINTER_ACCESS_DENIED");
```

Comment:
interesting, the error message make me realise there is a bug: it should be msg.sender === _admin and message about admin too

Reply:
good catch.